### ISSUE TITLE
React and Accessibility Webinar

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#873]
## Title
>React and Accessibility.

### Description
>JavaScript frameworks are becoming the norm for creating powerful, fast, and adaptive web sites. One of the most popular frameworks is React. With the ability to create reusable components, a virtual DOM, and a massive support community behind it, React has advantages that are seemingly limitless.

However, React has also developed a stigma within the accessibility community that web applications created in React are not accessible. The fact is though that React does, in fact, have a lot of accessibility that is built into the framework, and that it also has a large community of accessible add-ons that can help you achieve accessible components.

In this webinar, Mark Steadman will cover some tips and tricks that can help you achieve the most accessible react application possible!.

### URL
>https://accessibility.deque.com/react-accessibility-webinar.

### Date
>2020-02-04.

### Type
>"Webinar".

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/f781b7ba814171164b9d6f0dfa082635539765e2. Thank you!
